### PR TITLE
DOC: Add disambiguation article

### PR DESCRIPTION
**Pull request by bzah**
This article is a start to clarify the scope of icclim vis-à-vis xclim.

### Pull Request for the issue #101
- [NA] These changes were tested with real data and unit tests cover them as well.
- [x] The relevant documentation has been added (or updated).
- [x] A short description of the changes has been added to doc/source/references/release_notes.rst

### Describe the changes you made
Wrote an article to explain why icclim exists and may be relevant in 2022.

### CODE REVIEW

**Comment by pagecp** [comment]
I like very much the description and explanation of the differences between the two libraries. We could also mention that the scope of xclim is larger. Also when the C3S integration is done we should mention it.

File: doc/source/explanation/xclim_and_icclim.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. _clix-meta: https://github.com/clix-meta/clix-meta
+.. _xclim: https://github.com/Ouranosinc/xclim
+
+Disambiguation on xclim and icclim
+==================================
+
+On a first glance it seems `xclim`_ and icclim serve the same purpose but the difference is in the details.
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:7]:
```suggestion
At first glance it seems `xclim`_ and icclim serve the same purpose but the difference is in the details.
```

File: doc/source/explanation/xclim_and_icclim.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+With version 5 of icclim, xclim became a building block of icclim. xclim handles some the core features, notably
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:8]:
```suggestion
With version 5 of icclim, xclim became a building block of icclim. xclim handles some of the core features, notably
```

File: doc/source/explanation/xclim_and_icclim.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+the calculation climate indices. Beside, we also make use of xclim capabilities to handle i/o units and to check
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:9]:
```suggestion
the calculation of climate indices. Beside, we also make use of xclim capabilities to handle i/o units and to check
```

File: doc/source/explanation/xclim_and_icclim.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+input data and metadata validity.
+
+This relationship between the two libraries means they can benefit from one another.
+While developing icclim v5, we decided to integrate as much as possible relevant features in xclim directly, instead of
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:13]: syntax unclear

File: doc/source/explanation/xclim_and_icclim.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+keeping them on our code base. This is for example the case for the bootstrapping of percentile which was a
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:14]:
```suggestion
keeping them on our code base. This is for example the case for the bootstrapping of percentiles, which was a
```

File: doc/source/explanation/xclim_and_icclim.rst [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+collaborative work from both parties.
+This way, xclim gains new features and users through us and we can rely on all the
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:16]:
```suggestion
This way, xclim gains new features and users, and we can rely on all the
```

File: doc/source/explanation/xclim_and_icclim.rst [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+their repository.
+
+This joint effort helped tremendously to create what icclim is today and we believe it strengthens the collaborative
+ecosystem of geo sciences.
+
+--------
+
+On the other side, we had to clarify what would be the purpose of icclim without its internal climate index computation
+feature.
+Thus, icclim is now a library which wraps index calculation in a familiar API while **pre-processing** inputs and dress
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:27]: dress ?

File: doc/source/explanation/xclim_and_icclim.rst [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+output with **metadata**.
+
+Pre-processing includes:
+
+- Handling of multiple input formats (netcdf files, xarray.Dataset, text file (TBD)).
+- Simple automated data chunking, then fed to Dask.
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:33]:
```suggestion
- Simple automated data chunking
```

File: doc/source/explanation/xclim_and_icclim.rst [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+Beside, icclim still provides a way to write **user defined indices** using a simili JSON data structure
+(actually a python dictionary) with ``icclim.index(user_index=...)``.
+The metadata is however not as rich as with ECA&D indices.
+
+One of the goal of icclim is also to provide an API which require zero knowledge of xarray letting user new to python
+and its ecosystem reliably compute climate indices.
+This is one of the reason why icclim exposes a single entry point for all it's features with ``icclim.index``.
+It means users new to xarray might prefer icclim while experts could find xclim more convenient.
+
+xclim scope is larger than the one of icclim. xclim add metadata to the computed index in multiple languages (currently
```
Comment (huard) [inline doc/source/explanation/xclim_and_icclim.rst:56]:
```suggestion
xclim' scope is larger than the one of icclim. xclim add metadata to the computed index in multiple languages (currently
```

**Review by huard — APPROVED** [review]
Nice !

**Comment by bzah** [comment]
Thanks a lot for the review @huard and @pagecp 🚀

**Pull request merged**